flowback/poll/services/poll.py

```diff
@@ -137,6 +140,14 @@ def poll_create(*, user_id: int,
                                   category='poll_schedule',
                                   message=f'Poll {poll.title} has finished, group schedule has been updated',
                                   related_id=poll.id)
+    
+    topic = message_channel_topic_create(
+        channel_id=group_user.group.chat_id,
+        topic_name=f'Poll.{poll.id}',
+    )
+
+    poll.message_channel_topic = topic
```

Comment:
Unnessesary, if you wish to change this, check flowback/poll/models.py line 240 c:

Reply:
Thanks @Gofven 
Is the signal working properly, then we can use that signal.

Reply:
Yes it does work properly!